lib/generators/ruby_ui/component_generator.rb

```diff
@@ -74,8 +74,8 @@ def controller_info(controller_file)
         # Build the new import path
         new_import_path = new_import_path("./#{relative_path.dirname}/#{file_name}")
 
-        # Create the registration name by dasherizing the component name and prefixing with 'rbui--'
-        registration_name = "rbui--#{component_name.dasherize}"
+        # Create the registration name by dasherizing the component name and prefixing with 'ruby_ui--'
+        registration_name = "ruby_ui--#{component_name.dasherize}"
```

Comment:
i think `rbui--something` should be replaced with `ruby-ui--something` instead of `ruby_ui--something`

Reply:
@SethHorsley , I do agree with @pierry01 here. It will be violating stimulus conventions.

Reply:
Yes good catch. I can fix it in another pr.